.github/workflows/unit-tests-check.yml

```diff
@@ -0,0 +1,39 @@
+name: Ensure Unit Tests Running
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run_tests:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Required Packages
+      run: |
+        sudo apt-get update && sudo apt-get install libxmlsec1-dev
+        sudo systemctl start mongod
+
+    - name: Install pip
+      run: pip install -r requirements/edx/pip-tools.txt
+
+    - name: Install Python Dependencies
+      run: |
+        pip install setuptools wheel
+        pip install -r requirements/edx/development.txt
```

Comment:
I think `pip install -r requirements/edx/testing.txt -r requirements/edx/django.txt` would still work and save some time by installing fewer packages.

Reply:
changed to `requirements/edx/testing.txt`

Reply:
Working on caching deps in https://openedx.atlassian.net/browse/BOM-2591, which will optimize our runtime.